anatomy/tracks/concept-exercises.md

```diff
@@ -30,25 +30,31 @@ Concept Exercise metadata is defined in the `exercises.concept` key in the [conf
 
 Each Concept Exercise has its own directory within the track's `exercises/concept` directory. The name of the Concept Exercise directory must match the `slug` property of the Concept Exercise, as defined in the [config.json file](./config-json.md#concept-exercises).
 
-A Concept Exercise is comprised of three types of files:
+A Concept Exercise has three types of files.
```

Comment:
```suggestion
A Concept Exercise has three types of files:
```

I think we still want the colon to show that the headings are linked to this sentence.